### PR TITLE
test: refactor some tests

### PR DESCRIPTION
**Pull request by Flrande**

### CODE REVIEW

File: tests/basic.spec.ts [hunk 1]
```diff
@@ -88,7 +88,7 @@ test('A open and edit, then joins B', async ({ browser, page: pageA }) => {
   const room = await enterPlaygroundRoom(pageA);
   await initEmptyParagraphState(pageA);
   await focusRichText(pageA);
-  await pageA.keyboard.type('hello');
+  await pageA.keyboard.type('hello', { delay: 50 });
```
Comment (doodlewind) [inline tests/basic.spec.ts:91]: Use a `type` method

**Comment by doodlewind** [comment]
Can we move `virgo.spec.ts` back to `packages/virgo`? It should be package level test (see https://github.com/toeverything/blocksuite/discussions/756#discussioncomment-4713457).

**Comment by Flrande** [comment]
I think so

File: tests/format-quick-bar.spec.ts [hunk 2]
```diff
@@ -548,6 +548,12 @@ test('should format quick bar not show at readonly mode', async ({ page }) => {
 
 async function scrollToTop(page: Page) {
   // await page.mouse.wheel(0, -1000);
+
+  // When we type some characters, the character will be rendered asynchronously and then
+  // the editor will be focused asynchronously,
+  // which will cause the scroll container to scroll to the cursor position. That's why we need to
+  // wait for a while before scrolling.
+  await page.waitForTimeout(2000);
```
Comment (lawvs) [inline tests/format-quick-bar.spec.ts:556]: Is it necessary to wait 2s, which is too long?

**Review by lawvs — APPROVED** [review]

**Pull request merged**